src/db/migrations/20190403212306-create-article.js

```diff
@@ -42,6 +42,10 @@ export default {
         type: Sequelize.TEXT,
         allowNull: false,
       },
+      tags: {
```

Comment:
Nice work @emmsdan. However, since you are using a join table for articles and tags, this column is not needed

Reply:
yeah, I have removed that.  thanks